src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const perDayCost = 40;
+  const manyDaysOff = 50;
+  const fewDaysOff = 20;
+  const manyDaysCount = 7;
+  const fewDaysCount = 3;
+
+  if (days >= manyDaysCount) {
+    return (days * perDayCost - manyDaysOff);
+  }
+
+  if (days >= fewDaysCount && days < manyDaysCount) {
```

Comment:
```suggestion
  else if (days >= fewDaysCount) {
```
This check is redundant.
Also, use `else if` construction